analysis/report.ipynb

```diff
@@ -45,14 +46,12 @@
    "source": [
     "# OpenSAFELY-TPP Database Schema\n",
     "\n",
-    "# Introduction\n",
-    "\n",
     "This report displays the schema of the OpenSAFELY-TPP database.\n",
     "It is part of the OpenSAFELY platform's technical documentation\n",
     "and is published at <https://reports.opensafely.org/>.\n",
     "\n",
     "The schema is read from the `OpenSAFELYSchemaInformation` table,\n",
-    "which is refreshed at the same time as the core SystmOne primary care datasets.\n",
+    "which is refreshed at the same time as the core SystemOne primary care datasets.\n",
```

Comment:
I dont think this was a typo, see https://tpp-uk.com/

> "SystmOne, our core product, makes the 50 million electronic health records ... "

Reply:
Doh! Thanks.